src/main/java/org/nantipov/utils/wordhugopress/config/job/convertposts/ConvertPostsReaderConfig.java

```diff
@@ -25,25 +25,55 @@
 @Configuration
 public class ConvertPostsReaderConfig {
 
-    private static final String SQL_QUERY_POSTS =
-            "SELECT\n" +
+    private static final Set<String> DRAFT_STATUSES = ImmutableSet.of("draft", "auto-draft");
+
+    static final String BEAN_NAME = "postsReader";
+
+    private final SourcesSettings sourcesSettings;
+    
+    public ConvertPostsReaderConfig(SourcesSettings sourcesSettings) {
+        this.sourcesSettings = sourcesSettings;
+    }
+    
+    private static String TABLE_PREFIX;
+
+    private static String SQL_QUERY_POSTS; 
+
+    @Bean(BEAN_NAME)
+    public ItemReader<Post> reader() {
+        return CompositeItemReader.of(
+                sourcesSettings.getSources()
+                               .entrySet()
+                               .stream()
+                               .map(entry -> reader(entry.getKey(), entry.getValue()))
+                               .collect(Collectors.toList())
+        );
+    }
+
+    private ItemReader<Post> reader(String sourceName, SourcesSettings.Source source) {
+        if (source.getTags() != null && !source.getTags().isEmpty())
+            TABLE_PREFIX = source.getWordpressTablePrefix();
+        else
+            TABLE_PREFIX = "wp";
```

Comment:
1. Common code format for this repository. Please use curly brackets for `if` and narrow constructions even for single statements blocks (e.g. `{...} else {...}`).
2. Assignment to the `static` variable from non-static method. Should not be done in regular cases.
3. How does table prefix functionality correlate to the presence of absence of configured `tags`?
4. Probably, this logic was meant in some way to handle the default value of the table prefix. If so, please define `"wp"` as default value in the `SourcesSettings.Source` configuration class.

Reply:
For point 3, That's my fault for copying getTags statement without renaming to getWordpressTablePrefix.

It's better to define default value in `SourcesSettings.source`?

Reply:
Yes, I think that defining default value in the configuration class would make it quite clear. There is no logic required for handling this use-case, we simply read this from configuration and use the value what was defined or redefined.

Something like this.
```java
private String wordpressTablePrefix = "wp";
```

Or do you worry about the situation when user might enter some invalid values (e.g. empty, special characters)? For this case, we can think about introduction of the properties validation step. However, I would move this into separate issue/pull-request for now.

Reply:
I've revised that code in my new commit.